examples/GUI/GUIGlobe.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Generated On: 2015-10-5
+ * Class: GUIGlobe
+ * Description: Classe pour créer un menu.
+ */
+
+
+function GUIGlobe() {
```

Comment:
Why *Globe* suffix? This is a helper class to build a GUI to display/manipulate layers, so what do you think of `LayerGUI` ?